scipy/optimize/tests/test__basinhopping.py

```diff
@@ -106,10 +113,9 @@ def setUp(self):
 
         Run tests based on the 1-D and 2-D functions described above.
         """
-        self.x0 = (1.0, [1.0, 1.0])
-        self.sol = (-0.195, np.array([-0.195, -0.1]))
-        self.upper = (3., [3., 3.])
-        self.lower = (-3., [-3., -3.])
+        self.x0 = (1.0, [1.0, 1.0], [0.0, 0.0])
+        self.sol = (-0.195, np.array([-0.195, -0.1]), np.array([2.0, -1.0]))
```

Comment:
Apparently, you don't use the extra item in `self.sol`. So better drop it  (or use it). Also, maybe inline these data within the test method they're used at, since they seem to be used only once.